expenses/templates/expenses/report_run_csv.csv

```diff
@@ -0,0 +1 @@
+    {{ report_html }}
```

Comment:
Those four spaces appear in the file, which means the first column gets extra four spaces in front. It would be best to remove this file and not use templating altogether when rendering CSVs.